backend/hitas/services/reports.py

```diff
@@ -304,3 +317,91 @@ def build_regulated_housing_companies_report_excel(
     resize_columns(worksheet)
     worksheet.protection.sheet = True
     return workbook
+
+
+def build_unregulated_housing_companies_report_excel(
+    housing_companies: list[HousingCompanyWithUnregulatedReportAnnotations],
+) -> Workbook:
+    workbook = Workbook()
+    worksheet: Worksheet = workbook.active
+
+    column_headers = UnregulatedHousingCompaniesReportColumns(
+        housing_company_name="Yhtiö",
+        postal_code="Postinumero",
+        completion_date="Valmistumispäivä",
+        release_date="Vapautumispäivä",
+        released_by_ploy_department="Vapautettu tontit yksikön toimesta?",
+        apartment_count="Asuntojen lukumäärä",
+    )
+    worksheet.append(column_headers)
+
+    for housing_company in housing_companies:
+        worksheet.append(
+            UnregulatedHousingCompaniesReportColumns(
+                housing_company_name=housing_company.display_name,
+                postal_code=housing_company.postal_code.value,
+                completion_date=housing_company.completion_date,
+                release_date=housing_company.release_date,
+                apartment_count=housing_company.apartment_count,
+                released_by_ploy_department=(
+                    housing_company.regulation_status == RegulationStatus.RELEASED_BY_PLOT_DEPARTMENT
+                ),
```

Comment:
Here also I think it would be clearer to only display TRUE values (clearer at a quick glance)

Reply:
Made this value "X" or "" instead.